spec/services/organisations_service_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe OrganisationsService do
+  describe '#find_each' do
+    it 'queries the search API for all organisations' do
+      expected_params = {
+        query: { filter_format: 'organisation' },
+        fields: %w(slug title)
+      }
+      expect(Clients::SearchAPI).to receive(:find_each).with(expected_params)
+
+      subject.find_each {}
+    end
+
+    it 'yields the response' do
+      result = []
+      allow(Clients::SearchAPI).to receive(:find_each).and_yield(:a).and_yield(:b)
+      subject.find_each { |value| result << value }
+
+      expect(result).to eq([:a, :b])
```

Comment:
I suggest `match_array` here and also in [this](https://github.com/alphagov/content-performance-manager/blob/b979b439c41b39e4602409febc7c99fd26cc68df/spec/services/content_items_service_spec.rb#L26) as well.. see [this](https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers)